src/lib/Elf.cc

```diff
@@ -66,7 +66,8 @@ Elf::Elf(std::string path) {
     }
   }
 
-  processImage_ = new char[processImageSize_];
+  *imagePointer = (char*)malloc(processImageSize_ * sizeof(char));
+  processImage_ = *imagePointer;
```

Comment:
Shouldn't we be getting rid of this copy as well?

Reply:
Removed. PTAL.